packages/fluentui/react-northstar-prototypes/package.json

```diff
@@ -0,0 +1,70 @@
+{
+  "name": "@fluentui/react-northstar-prototypes",
+  "description": "A set of prototypes built using @fluentui/react-northstar",
+  "version": "0.51.0",
+  "bugs": "https://github.com/microsoft/fluentui/issues",
+  "dependencies": {
+    "@babel/runtime": "^7.10.4",
+    "@fluentui/accessibility": "^0.51.0",
+    "@fluentui/code-sandbox": "^0.51.0",
+    "@fluentui/docs-components": "^0.51.0",
+    "@fluentui/keyboard-key": "^0.2.12",
+    "@fluentui/react-bindings": "^0.51.0",
+    "@fluentui/react-component-event-listener": "^0.51.0",
+    "@fluentui/react-component-ref": "^0.51.0",
+    "@fluentui/react-icons-northstar": "^0.51.0",
+    "@fluentui/react-northstar": "^0.51.0",
+    "tslib": "^1.10.0",
+    "classnames": "^2.2.6",
+    "copy-to-clipboard": "^3.2.0",
+    "csstype": "^2.6.7",
+    "faker": "^4.1.0",
+    "formik": "2.0.3",
+    "lodash": "^4.17.15",
+    "moment": "^2.18.1",
+    "react": "16.8.6",
+    "react-custom-scrollbars": "^4.2.1",
+    "react-dom": "16.8.6",
+    "react-hook-form": "^5.7.2",
+    "react-textarea-autosize": "7.0.0",
+    "react-virtualized": "^9.21.1",
+    "react-window": "^1.8.6"
```

Comment:
Let's try to cleanup deps that are not required in this package

Reply:
I bet you a beer this is clean :D I ran depcheck